apps_download/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=============================
+Product Download for Appstore
+=============================
+
+Providing facility for the maintain dependency of Module at level of product
+and creating Zip file with all dependent modules
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/249/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/apps-store/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Ruchir Shukla <ruchir@bizzappdev.com> BizzAppDev https://www.bizzappdev.com
```

Comment:
Add my name:
Eric Caudal <eric.caudal@elico-corp.com>

Reply:
```
* Ruchir Shukla <ruchir@bizzappdev.com>
* Georg Nottere <georg.notter@agenterp.com>
```

Reply:
Done

Reply:
Please update using the standard format